data/RTTR/campaigns/roman/MISS206.lua

```diff
@@ -218,9 +218,12 @@ function addPlayerBld(p, onLoad)
             nil, nil
         )
 
-        if(p == 2) then
-            if onLoad then return end
+        if onLoad then return end
 
+        if(p == 1) then
+            rttr:GetPlayer(p):PlaceHQ(112, 82)  -- !SET_HOUSE 24, 112, 82
```

Comment:
just to make sure, if someone calls this multiple times for the same player, or even if loading, the game doesn't break does it?

Reply:
And another question, does this work in multiplayer as well? Since I really love this feature - one could create a map with 10 possible starting points, assigning them "randomly" (not random but by some kind of algorithm to use a spot for a player) or in a campaign where enemies can come from different sides, each time you play the mission

Reply:
The PlaceHQ call is ignored if there is already an HQ. The map data takes precedence because it is loaded before any of these PlaceHQ calls are made. If you put two HQs for a player from LUA then only the first one will be placed. This needs documenting.

I didn't test multiplayer, but it would only work with "broken" maps, because it is designed for "broken" maps :) You would have to create a map with missing player positions and then it should work, but I didn't check. For the use case you mentioned I think a different function should be designed, such as `AssignHQToPlayer` or something. Then the map would (correctly) define positions, but LUA would shuffle them around players.